### ISSUE TITLE
Module not found: Error: Can't resolve 'vuejs-storage' in ...

### ISSUE THREAD

**Issue by wiliam-paradox** [maple3142/vuejs-storage#38]
VueJS Storage started to fail some hours ago, after 3.0.2 commit.

When doing an `asset-prod` it outputs:

```
 ERROR  Failed to compile with 1 errors                                              1:40:47

This dependency was not found:

* vuejs-storage in ./node_modules/@xxx/mm-component-lib/src/install.js

To install it, you can run: npm install --save vuejs-storage
 4 assets

ERROR in ./node_modules/@xxx/mm-component-lib/src/install.js
Module not found: Error: Can't resolve 'vuejs-storage' in '...'
 @ ./node_modules/@xxx/mm-component-lib/src/install.js 32:0-41 131:12-24
 @ ./node_modules/@xxx/mm-component-lib/index.js
 @ ./resources/js/app.js
 @ multi ./resources/js/app.js ./resources/sass/app.scss
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! control-center@1.0.0 asset-production: `cross-env NODE_ENV=production node_modules/
webpack/bin/webpack.js --no-progress --hide-modules --config=node_modules/laravel-mix/setup/
webpack.config.js`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the control-center@1.0.0 asset-production script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output
above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\xxx\AppData\Roaming\npm-cache\_logs\2020-08-25T23_40_47_576Z-debug.
log
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

Process finished with exit code 2
```

**Comment by maple3142** [comment]
Fixed in v3.0.3

**Comment by wiliam-paradox** [comment]
Broken again in 3.1.0. Same issue.
